### PR TITLE
chore: update thrift queries

### PR DESCRIPTION
**Pull request by amaanq**
I recently added support for a large extension of thrift, this PR addresses the query updates for that

### CODE REVIEW

**Comment by amaanq** [comment]
always forgetting lockfile change... :slightly_smiling_face:

**Comment by amaanq** [comment]
amended for v0.3.0 being published

File: queries/thrift/highlights.scm [hunk 1]
```diff
@@ -57,6 +60,9 @@
 ; Attributes
 
 (annotation_identifier) @attribute
+(exception_modifier) @attribute
+(field_modifier) @attribute
+(function_modifier) @attribute
```
Comment (theHamsta) [inline queries/thrift/highlights.scm:65]: are modifiers annotations? or `@type.qualifier`?

**Review by theHamsta — APPROVED** [review]

File: queries/thrift/highlights.scm (hunk 1 above)
Comment (amaanq) [reply queries/thrift/highlights.scm:65]: not entirely sure what `@type.qualifier` is, but they are keywords before a function like async, before a field like required, optional

Comment (theHamsta) [reply queries/thrift/highlights.scm:65]: yes, then not `@attribute`. Attribute is for annotations that are not actually necessary in the code but can be added for additional meta info the compiler/refection or alike (originally for C++ attributes) or those Go serialization tags. `@type.qualifier` and `@storageclass` are typically used for stuff like `const`, `async`, `volatile` or just `@keyword`.

Comment (amaanq) [reply queries/thrift/highlights.scm]: gotcha, thanks! fixed now

**Review by theHamsta — APPROVED** [review]

**Pull request merged**